src/models/account.cpp

```diff
@@ -507,6 +526,185 @@ int Account::importFromCSV(const std::string& path)
     return imported;
 }
 
+int Account::importFromOFX(const std::string& path)
+{
+    int imported{ 0 };
+    unsigned int nextId{ getNextAvailableTransactionId() };
+    const std::regex xmlTagRegex("^<([^/>]+|/STMTTRN)>(?:[+-])?([^<]+)");
```

Comment:
const isn't really needed here.

also for variable initialization use `{}`: `std::regex xmlTagRegex{ "^<([^/>]+|/STMTTRN)>(?:[+-])?([^<]+)" };

Reply:
For the variable initialization, I have to say I followed the documentation for that one